README.md

```diff
@@ -110,13 +110,11 @@ From 2019-02-01 to 2019-03-31 : 13h1m48s
 
 
 ## Why did I make it?
-https://github.com/kimmobrunfeldt/git-hours
-
-To install the git hours command through the above repository uri, you need to install node.js and dependencies.
-I thought that it would be good if there was a tool that can be executed immediately after downloading.
-
-reference model : https://github.com/kimmobrunfeldt/git-hours#how-it-works
-
-The code I wrote before the first commit was set to a model that basically add 1 hour, including programming immersion time.
-
-I like projects that are carried out by telecommuting. For me, it is more convenient than https://toggl.com service. It is also a trust model compared to the toggle service.
+I'm interested in working from home space.
+I want to calcurate working time.
+I feel that [toggle-style program](https://toggl.com) is high-maintenance tool. because users have to pushing button every time the work starts or ends.
+Mechanism of [kimmobrunfeldt's git-hours](https://github.com/kimmobrunfeldt/git-hours#how-it-works) looked reasonable to me.
+I tried to install via https://github.com/kimmobrunfeldt/git-hours, Because of the node.js dependency, It didn't work well on my computer.
```

Comment:
- However
- Because of the node.js dependency, It didn't work well on my computer.